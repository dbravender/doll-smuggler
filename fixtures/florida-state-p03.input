max weight: 190

available dolls:

name  weight  value
a         56     50 1
b         59     50 1
c         80     64 0
d         64     46 0
e         75     50 1
f         17      5 0
