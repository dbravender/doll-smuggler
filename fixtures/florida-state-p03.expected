packed dolls:

name   weight  value
a          56     50
b          59     50
e          75     50
